readme.md

```diff
@@ -1,151 +1,131 @@
+Install and run your own Renovate server. Access internal packages.
 
-## Star History
+- [Mend Renovate Community Self-Hosted](https://github.com/mend/renovate-ce-ee/tree/main/docs) (Free)
+- [Mend Renovate Enterprise](https://www.mend.io/mend-renovate/) (Paid plan)
 
-[![Star History Chart](https://api.star-history.com/svg?repos=renovatebot/renovate&type=Date)](https://star-history.com/#renovatebot/renovate&Date)
+### Other ways to run Renovate
 
-## The Renovate Approach
+If you can’t use a pre-built job scheduling system, or want to build your own, the following options are available:
 
-We believe everyone benefits from automation, whether it's a little or a lot.
-This means that Renovate:
+#### Run Renovate on your Pipeline
 
-- Adapts to your workflow
-- Allows you to configure its behavior
-- Will autodetect settings where possible
+Mend provides a _**GitHub Action**_ and _**GitLab Runner**_ to make it easier to run Renovate as a CI pipeline job.
 
-## Using Renovate
+- GitHub Action: [renovatebot/github-action](https://github.com/renovatebot/github-action).
+- GitLab Runner: [Renovate Runner project](https://gitlab.com/renovate-bot/renovate-runner/)
+- AzureDevOps action: [Renovate Me extension](https://marketplace.visualstudio.com/items?itemName=jyc.vsts-extensions-renovate-me)<br>
+  _Note: This extension is created and maintained personally by a Renovate developer/user so support requests relating to the extension itself cannot be answered directly in the main Renovate repository._
+- Custom pipeline: You can create a custom pipeline with a **yml** definition that triggers **npx renovate**. [More details on how to configure the pipeline](https://docs.renovatebot.com/modules/platform/azure/).
 
-Get started with Renovate by checking out our [tutorial](https://github.com/renovatebot/tutorial).
+#### Run Renovate CLI
 
-### GitHub
+There are several ways to run the Renovate CLI directly.
+See docs: [Running Renovate](https://docs.renovatebot.com/getting-started/running/) for all options.
 
-We recommend that you use the Mend Renovate App.
-Install [the Mend Renovate App](https://github.com/apps/renovate) now.
-[More details on the Mend Renovate App installation](https://docs.renovatebot.com/getting-started/installing-onboarding/).
+**Targets: all platforms**
 
-### Azure DevOps
+## Docs
 
-There are two ways to run Renovate on Azure DevOps:
+### More about Renovate
 
-- Renovate Me extension
-- Custom pipeline
+- Renovate basics
+  - [Why use Renovate](https://docs.renovatebot.com/#why-use-renovate)
+  - [What does it do? / How does it work?](https://docs.renovatebot.com/key-concepts/how-renovate-works/)
+  - [Who is using it?](https://docs.renovatebot.com/#who-uses-renovate)
+- Supported platforms and languages
+  - [Supported platforms](https://docs.renovatebot.com/#supported-platforms)
+  - [Supported languages / package managers](https://docs.renovatebot.com/modules/manager/)
+- Advanced renovate usage
```

Comment:
```suggestion
- Advanced Renovate usage:
```

Reply:
I don't really like adding colons to a list heading.